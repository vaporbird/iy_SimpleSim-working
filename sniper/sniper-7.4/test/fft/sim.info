{'cmdline': ['../../run-sniper',
             '-v',
             '-n',
             '1',
             '-c',
             'gainestown',
             '--roi',
             '--',
             './fft',
             '-p',
             '1'],
 'git_revision': '',
 'host': 'vanyo',
 'pin_version': 'unknown',
 'rusage': (1.996211,
            0.023717,
            30760,
            0,
            0,
            0,
            6103,
            0,
            0,
            0,
            144,
            0,
            0,
            0,
            4626,
            113),
 'snipercmd': ['bash',
               '-c',
               '/home/vanyo/sniper/sniper-7.4/lib/sniper -c /home/vanyo/sniper/sniper-7.4/config/base.cfg --general/total_cores=1 --general/output_dir=/home/vanyo/sniper/sniper-7.4/test/fft --config=/home/vanyo/sniper/sniper-7.4/config/nehalem.cfg --config=/home/vanyo/sniper/sniper-7.4/config/gainestown.cfg -g --general/magic=true -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -g --traceinput/stop_with_first_app=false -g --traceinput/enabled=true -g --traceinput/emulate_syscalls=true -g --traceinput/num_apps=1 -g --traceinput/trace_prefix=/tmp/tmpqZgcZg/run_benchmarks'],
 't_elapsed': 5.344186067581177,
 't_start': 1657292032.880589,
 'tracecmds': [['/home/vanyo/sniper/sniper-7.4/record-trace',
                '-o',
                '/tmp/tmpqZgcZg/run_benchmarks',
                '-v',
                '--roi',
                '-e',
                '1',
                '-s',
                '0',
                '-r',
                '1',
                '--follow',
                '--routine-tracing',
                '--',
                './fft',
                '-p',
                '1']],
 'user': 'vanyo',
 'vmem': 42512384L}
