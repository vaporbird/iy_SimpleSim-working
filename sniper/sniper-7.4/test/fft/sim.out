                                     | Core 0    
  Instructions                       |    1126892
  Cycles                             |    1311996
  IPC                                |       0.86
  Time (ns)                          |     493232
  Idle time (ns)                     |     214952
  Idle time (%)                      |      43.6%
Branch predictor stats               |           
  num correct                        |      31311
  num incorrect                      |        736
  misprediction rate                 |      2.30%
  mpki                               |       0.65
TLB Summary                          |           
  I-TLB                              |           
    num accesses                     |     104261
    num misses                       |         10
    miss rate                        |      0.01%
    mpki                             |       0.01
  D-TLB                              |           
    num accesses                     |     660752
    num misses                       |         14
    miss rate                        |      0.00%
    mpki                             |       0.01
  L2 TLB                             |           
    num accesses                     |         24
    num misses                       |         24
    miss rate                        |    100.00%
    mpki                             |       0.02
Cache Summary                        |           
  Cache L1-I                         |           
    num cache accesses               |     104261
    num cache misses                 |        154
    miss rate                        |      0.15%
    mpki                             |       0.14
  Cache L1-D                         |           
    num cache accesses               |     660752
    num cache misses                 |       1929
    miss rate                        |      0.29%
    mpki                             |       1.71
  Cache L2                           |           
    num cache accesses               |       2191
    num cache misses                 |        606
    miss rate                        |     27.66%
    mpki                             |       0.54
  Cache L3                           |           
    num cache accesses               |       1023
    num cache misses                 |        606
    miss rate                        |     59.24%
    mpki                             |       0.54
DRAM summary                         |           
  num dram accesses                  |        605
  average dram access latency (ns)   |      65.37
  average dram queueing delay        |      11.94
  average dram bandwidth utilization |      1.00%
Coherency Traffic                    |           
  num loads from dram                |        429
  num loads from dram cache          |          0
  num loads from remote cache        |          0
